### DISCUSSION TITLE
功能需求：兼容 Rbatis GoMybatis 的生成器工具，类似于MBG

### DISCUSSION THREAD

**Discussion by ghost** [rbatis/rbatis#267] [General]
Mybatis 有个很强大的工具，名为：MBG

 https://mybatis.org/generator/quickstart.html

就算你是海量数据表，用这个工具，也能快速帮你生成数据表xml文件和各种Entity.

目前Rbatis和GoMybatis 是没有这样的生成工具可以用的。

如果能补一个这样的工具， 或者说：直接食用MyBatis生成的xml文件（可能还不太够，因为Java生成的有适用于Java的一堆Entity 等 java源码文件）
Go和Rust也是需要这样的一些文件的。

**Comment by ghost** [comment]
Rust 里面的SeaORM 好像提供了接近的方案。

![image](https://user-images.githubusercontent.com/278153/165695229-3dc3cf14-5071-41de-9e16-effd885664e5.png)


代码生成器，自动生成Entity 文件源代码，这样开发者直接import 就能用

**Comment by Jzow** [comment]
你指的是 通过配置 自动映射实体吗

**Comment by longzou** [comment]
可以去看看这个： 
https://gitee.com/poethxp/rbatis-generator/
或者
https://github.com/longzou/rbatis-generator

专门用于为rbatis来生成代码的工具。
